### DISCUSSION TITLE
Lenovo Slim 7 Pro 14ARP6

### DISCUSSION THREAD

**Discussion by plastic-racer** [BartoszCichecki/LenovoLegionToolkit#773] [General]
I know the slim series isnt the same as legion, but is a decent powerhouse slimline version. Can this be added as a supported model? If so what is needed to help with anything needed to make this happen? Looking for automatically change power plans to work it does not seem to work, and do not see hybrid gpu option

**Comment by BartoszCichecki** [comment]
I don't have the time nor the resources to support more models. I am open to contributions, but helping with testing is not enough. If someone wants to reverse engineer how other models handle these features and implements it, it might be added in the future.
